UnitySDK/Assets/ML-Agents/Examples/BananaCollectors/Scripts/BananaAgent.cs

```diff
@@ -28,7 +28,7 @@ public class BananaAgent : Agent
     public Material frozenMaterial;
     public GameObject myLaser;
     public bool contribute;
-    private RayPerception rayPer;
+    private RayPerception3D rayPer;
```

Comment:
This is going to break people's existing code

Reply:
Every major release involves breaking changes.